testutil/server.go

```diff
@@ -165,7 +165,8 @@ func NewTestServer(t TestingT) *TestServer {
 // an optional callback function to modify the configuration.
 func NewTestServerConfig(t TestingT, cb ServerConfigCallback) *TestServer {
 	if path, err := exec.LookPath("consul"); err != nil || path == "" {
-		t.Skip("consul not found on $PATH, skipping")
```

Comment:
Hmm - you probably are thinking that you are running all these tests so hard fail makes sense. If you don't have a local consul then you are probably only running some subset of tests you care about, so we probably don't need an env var unless this blows lots of places up in unexpected ways.